Вселенная RimWorld/cryptosleep_revival_briefing_trans.txt

```diff
@@ -0,0 +1,147 @@
+  * Окраинные миры: Планеты без сильного централизованного правительства и с низкой плотностью населения. Эти места, как правило, повисают вблизи индустриального уровня технологий или даже ниже. Поскольку они не приведены к единообразию центральным правительством, в них часто можно видеть множество взаимодествий между людьми разного технологического уровня, например потерпят крушение путешественики, либо древние общины натыкаются на усыпальницы криптосна этих путешествеников (при.пер. саркофаги есть, почему бы vault (по факту помещение, имеющее свод) в данном случае можно перевести как склеп, усыпальница). Планеты эти зачастую лежат на окраинах известного космоса, отсюда и название.
+  * Токсичные миры: Миры, уничтоженные загрязнением, химической или ядерной войной, но всё ещё пригодные для низкоуровневого обитания при адекватных технологиях.
+  * Остеклевшие миры (Glassworlds): Миры полностью уничтоженные высокоэнергетичным оружием массового поражения. Их поверхность «остеклилась» (прим.пер. витрифицирована), за что их прозвали «стеклянные шарики» (прим.пер. можно и калькой, "марблы"). Ядерное оружия не способно витрифицировать целиком планету, поэтому такой уровень разрушений редок. На некоторых из этих миров люди могут какое-то время находится на открытом воздухе без опасности умереть. Но никто больше инфузории не сможет надолго бросить здесь якорь жизни.
+  * Трансцендентные миры: Весьма натянуто называть эти сущности мирами, поскольку они больше походят на гигантские компьютеры, чем на планеты. Механика этих планет загадочна, но многие изучающие их считают трансценденты последствием решения правителя-архотека (sovereign archotech) слится воедино со всей планетой. Подробней об этом далее.
+  * Остальные миры: Помимо этих категорий, множество необычных планет пребывает в удивительных состояниях, созданных свойственным только им социальными и технологическими эволюциями. Учитывая масштаб и возраст вселеной, для формирования огромного количества крайне странных ситуаций существует такое же количество времени и пространства. (прим.пер. вселенная большая и старая, и возможно всё)
+
+    Ключевые технологии
+
+В этой вселенной присутствует неисчислимое количество новых технологий, но несколько ключевых, имеющих сильное влияние на последующую жизнь человечества в Млечном Пути стоят особняком.
+  * Технологии среднего мира: Все реальные технологии в истории Земли вплоть до 21-го столетия играют важную роль и в настоящий момент. Поскольку есть планеты с любым уровнем технологического развития, начиная с каменного века, есть и технологии от луков со стрелами и паровых двигателей до ядерных бомб и мобильных телефонов включительно на любой из них.
```

Comment:
>Since there are planets at every level of technological development from the stone age on up, there are technologies from bows and arrows to steam engines to nuclear bombs and cellphone all in use across various planets.

> Поскольку есть планеты с любым уровнем технологического развития, начиная с каменного века, есть и технологии от луков со стрелами и паровых двигателей до ядерных бомб и мобильных телефонов включительно на любой из них.

Поскольку существуют планеты с любым уровнем технологического развития, начиная с каменного века, на них применяются соответствующие технологии: от луков со стрелами и паровых двигателей до ядерных бомб и мобильных телефонов.